tests/src/test/scala/com/nvidia/spark/rapids/OrcQuerySuite.scala

```diff
@@ -26,10 +26,10 @@ import org.apache.hadoop.fs.FileUtil.fullyDelete
 import org.apache.hadoop.fs.Path
 import org.apache.orc.{OrcFile, StripeInformation}
 import org.apache.orc.impl.RecordReaderImpl
-
 import org.apache.spark.{SparkConf, SparkContext}
+
```

Comment:
Please put the spacing back the way it was before. The build is complaining about it.